### DISCUSSION TITLE
Best graphics card and CPU for Meshroom

### DISCUSSION THREAD

**Discussion by raffi-gracie** [alicevision/Meshroom#2000] [General]
Hi! I am looking to buy a computer for my research group to use Meshroom. Does Meshroom work well with newer CPU/GPUs such as the Intel Core i7 and NVIDIA GeForce RTX 4070? I am looking to buy the best laptop for running Meshroom (and similar software for photogrammetry) and would love some advice! Thanks!

**Comment by natowi** [comment]
"The binaries are built with CUDA-11 and are compatible with compute capability >= 3.5. 
The full resolution can now be computed on most of the standard GPUs."

https://github.com/alicevision/Meshroom/releases/tag/v2023.1.0

So the RTX 4070 should work nicely
